codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerics.kt

```diff
@@ -102,6 +107,17 @@ data class FlexibleClientGenerics(
         )
     }
 
+    override val boundsWithoutWhereClause = writable {
+        rustTemplate(
+            """
+            C: #{client}::bounds::SmithyConnector,
```

Comment:
Ideally, `bounds` above would incorporate `boundsWithoutWhereClause` rather than duplicating them. Something like:
```kotlin
rustTemplate("where #{bounds}", "bounds" to boundsWithoutWhereClause)
```